determined-ai/determined
# fix: change average_training_metrics default to true [DET-7240]

## Description

BREAKING CHANGE: Change the default value of `average_training_metrics` in the `optimizations` section of the experiment config to `true`.  The current `false` default leads to only the chief's training metrics being reported, which is surprising behavior.




## Test Plan

Manual testing:
- Verify that changing the default doesn't break selected experiments in `examples/`
- Perform modest benchmarking to verify that this default does not degrade performance to an appreciable degree.

Also verify that changing this default does not break existing tests.




## Commentary (optional)




## Checklist

See [Release Note](../docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
